### ISSUE TITLE
Specify detekt version from settings

### ISSUE THREAD

**Issue by ankushg** [detekt/detekt-intellij-plugin#125]
Sometimes I'm working on a project that doesn't use the latest version of detekt.

It would be great if the IDE plugin had a way to configure which version of detekt to use on a project, instead of tightly coupling the Plugin version with the Detekt version.

**Comment by cortinico** [comment]
Yup that sounds like a good addition. It will also allow us to decouple the release cycle of this project from upstream detekt
